static/script/application.js

```diff
@@ -93,16 +93,27 @@ require.def('antie/application',
                         }
                     }
 
+                    var syncStartUp = function() {
+                        self.run();
+                        self.route(device.getCurrentRoute());
+                    };
+
                     require([_layout.module], function (layout) {
-                        self.setLayout(layout, styleBaseUrl, imageBaseUrl, _layout.css, _layout.classes, [], function () {
-                            self.run();
-                            self.route(device.getCurrentRoute());
+                        self.setLayout(layout, styleBaseUrl, imageBaseUrl, _layout.css, _layout.classes, [], function(){
```

Comment:
Why is layout not in the syncStartUp function?

My natural inclination is that the responsibility of this function should be to provide only the async nature of startup.